Documentation/gettingstarted/cni-chaining-aws-cni.rst

```diff
@@ -31,18 +31,16 @@ done.
 
 .. include:: k8s-install-download-release.rst
 
-Generate the required YAML files and deploy them:
-
-.. code:: bash
-
-   helm template cilium \
-     --namespace kube-system \
-     --set global.cni.chainingMode=aws-cni \
-     --set global.masquerade=false \
-     --set global.tunnel=disabled \
-     --set global.nodeinit.enabled=true \
-     > cilium.yaml
-   kubectl apply -f cilium.yaml
+Deploy helm release:
+
+.. parsed-literal::
+
+   helm install cilium/cilium --version |CHART_RELEASE| \\
```

Comment:
I believe it must be `helm install cilium cilium/cilium ...`, at least that's what my helm 3.0.1 binary is telling me:

```
Usage:
  helm install [NAME] [CHART] [flags]
```

Reply:
Yep, I used helm v2 syntax. I have updated all occurrences to used v3 syntax.